features/delivery.feature

```diff
@@ -107,6 +107,7 @@ Feature: Delivery of errors
     Given I set the HTTP status code for the next request to 500
     And I run "AutoSessionScenario"
     And I wait to receive a session
+    And I switch to the web browser for 0 seconds
```

Comment:
Not sure if you have a commit coming, but this still needs to be removed at some point.